sycl/include/sycl/ext/intel/esimd/detail/types.hpp

```diff
@@ -19,7 +19,7 @@
 #include <sycl/half_type.hpp>
 
 #if defined(__ESIMD_DBG_HOST) && !defined(__SYCL_DEVICE_ONLY__)
-#define __esimd_dbg_print(a) std::cout << ">>> " << #a << "\n"
```

Comment:
LGTM, this thing is pretty useless anyway and we should consider removing it as part of host code removal

@turinevgeny You might want to add this to the list of stuff to remove with your host cleanup

Reply:
I have this change in my bag of changes, not published yet.